### PR TITLE
Make Mondrian view zoom in/out require primary modifier to be pressed. Consume event.

### PR DESCRIPTION
**Pull request by botwhytho**
Make Mondrian view zoom in/out require primary modifier to be pressed. Consume event.

### CODE REVIEW

**Comment by JurajKubelka** [comment]
Hi @botwhytho: Thank you for the pull request! Can we just use `event modifiers = BlKeyModifiers primary` instead of `event primaryModifierPressed`? I have the impression that the `primaryModifierPressed` method name is ambiguous. It could be interpreted as "is only primary modifier pressed?" or "is primary modifier pressed, optionally with any other modifier?"

In addition, I think we do not want to have the `BlMouseEvent >> primaryModifierPressed` method since there are many possible combinations of modifier. In other words, we do not want to convert all those combinations into `BlMouseEvent >> is...Pressed`. What do you think?

File: src/GToolkit-Mondrian-Phlow/GtPhlowMondrianView.class.st [hunk 1]
```diff
@@ -26,20 +26,23 @@ GtPhlowMondrianView >> asElementDo: aBlock [
...
-			do: [ :event | 
-				event isPrimarilyVertical
-					ifTrue: [ 
-					| eventYvector newZoomLevel scalingFactor | 
-					eventYvector := event vector y.
-					scalingFactor := (1/2).
-					newZoomLevel := (eventYvector positive ifTrue: [ mondrianElement zoomLevel * scalingFactor reciprocal ] ifFalse: [mondrianElement zoomLevel * scalingFactor]) round: 2.
-					mondrianElement zoomLevel: newZoomLevel ] ].
+			do: [ :event |
+				event primaryModifierPressed
```
Comment (JurajKubelka) [inline src/GToolkit-Mondrian-Phlow/GtPhlowMondrianView.class.st]: I suggest to use `event modifiers = BlKeyModifiers primary` on this line. The more detailed explanation is in this PR comment.

**Comment by botwhytho** [comment]
Okay, guess adding that method on mouse event might be 'premature optimization' if only used in one place (this changeset) currently. I've updated code to just test inline without that method

**Comment by botwhytho** [comment]
Closed other PR

**Comment by botwhytho** [comment]
Addressed your concern above. Let me know if you have any other comments.

**Review by JurajKubelka — APPROVED** [review]

**Pull request merged**